### PR TITLE
gptman: 0.8.3 -> 1.0.1

### PR DESCRIPTION
**Pull request by aaronjheng**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by figsoda** [comment]
the latest version seems to be 1.0.1

**Comment by figsoda** [comment]
there doesn't seem to be anything in the derivation, you might need to cli feature

File: pkgs/tools/system/gptman/default.nix [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
   };
 
-  cargoSha256 = "sha256-3PRGPZGymccRo9dtQZgMMEL29x+GiUkTzgc8uAB/ocQ=";
+  cargoHash = "sha256-voslPSh7n31cGTKaayKXomgiXWVTutuc4FxfnZUDejc=";
 
   buildInputs = lib.optional stdenv.isDarwin libiconv;
 
+  cargoBuildFlags = [ "--all-features" ];
+
+  cargoTestFlags = [ "--workspace" "--all-features" ];
```
Comment (figsoda) [inline pkgs/tools/system/gptman/default.nix:20]: This should be enough
```suggestion
  buildFeatures = [ "cli" ];
```

Comment (aaronjheng) [reply pkgs/tools/system/gptman/default.nix:20]: I'm OK with enabling `cli` feature only. But I think it's better to stay in line with upstream. See https://github.com/rust-disk-partition-management/gptman/blob/b0c3567c19c1b23f95b5de665a58d611301e3706/.github/workflows/release.yml#L25.

Comment (figsoda) [reply pkgs/tools/system/gptman/default.nix:20]: that's the only feature

File: pkgs/tools/system/gptman/default.nix [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
-    sha256 = "sha256-hI3F1E1vdbNDEeJ4FrU0EvR0t64svzUIpI6zaf0CquM=";
+    hash = "sha256-sDRnvF/XPXgWIIIrOmnEuktP8XvZxPahF2n4h8RCX+o=";
   };
 
-  cargoSha256 = "sha256-3PRGPZGymccRo9dtQZgMMEL29x+GiUkTzgc8uAB/ocQ=";
+  cargoHash = "sha256-voslPSh7n31cGTKaayKXomgiXWVTutuc4FxfnZUDejc=";
 
   buildInputs = lib.optional stdenv.isDarwin libiconv;
 
+  buildFeatures = [ "cli"];
```
Comment (figsoda) [inline pkgs/tools/system/gptman/default.nix:18]:
```suggestion
  buildFeatures = [ "cli" ];
```

**Review by figsoda — APPROVED** [review]

**Pull request merged**